sematic/ui/src/components/ScrollingLogView.tsx

```diff
@@ -111,26 +111,20 @@ export default function ScrollingLogView(props: {
   }, [getNext, hasPulledData]);
 
   return (
-    <Box
-      sx={{
-        mt: 5,
-        position: "relative",
-        left: 0,
-        top: 0,
-      }}
-    >
-      
+    <>
       <Box
         id={scrollerId}
         ref={scrollMonitorRef}
         sx={{
-          height: "400px",
-          my: 5,
+          height: 0,
+          mt: 5,
           pt: 1,
-          whiteSpace: "nowrap",
+          whiteSpace: "break-spaces",
           overflow: "hidden",
           overflowY: "scroll",
-          gridRow: 2,
+          width: `100%`,
+          lineBreak: 'anywhere',
+          flexGrow: 1
         }}
```

Comment:
Can we get a small right padding so that characters are not stuck against the scrollbar to the right?